ios/library/WhirlyGlobe-MaplyComponent/include/private/MaplyBaseInteractionLayer_private.h

```diff
@@ -205,10 +205,13 @@
 - (void)removeImageTexture:(MaplyTexture *__nonnull)tex changes:(WhirlyKit::ChangeSet &)changes;
 
 // Do a point in poly check for vectors we're representing
-- (NSArray *__nullable)findVectorsInPoint:(WhirlyKit::Point2f)pt;
 - (NSArray *__nullable)findVectorsInPoint:(WhirlyKit::Point2f)pt inView:(MaplyBaseViewController*__nullable)vc multi:(bool)multi;
 
+// Find MaplySelectableObjects at a screen point
 - (NSObject*__nullable)selectLabelsAndMarkerForScreenPoint:(CGPoint)screenPoint;
+- (NSMutableArray*__nullable)selectMultipleLabelsAndMarkersForScreenPoint:(CGPoint)screenPoint;
+- (NSMutableArray*__nullable)convertSelectedObjects:(std::vector<WhirlyKit::SelectionManager::SelectedObject>)selectedObjs;
```

Comment:
The collection should be passed by constant reference (`const .. &`) here to avoid making an unnecessary copy.

Reply:
Thank you @TimSylvester-mousebird for the review. I haven't a lot c++ experience, so please let me know if fix is not correct.